tests/libvmi/cloudinit.go

```diff
@@ -25,7 +25,10 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 )
 
-const cloudInitDiskName = "disk1"
+const (
+	cloudInitDiskName = "disk1"
+	BashHelloScript   = "#!/bin/bash\necho 'hello'\n"
```

Comment:
I do not think this has anything to do with cloud-init configuration.
It is used for the Cirros and Alpine factories and should not be needed by anything else unless you show the need for it.

The usages for the Fedora based images are not needed and most likely can be dropped (they are leftovers).

Reply:
To me, it is widely used outside of Cirros and Alpine factories and it is always used with cloud-init configuration.
So from my point of view, for how things are right now, it fits the cloud-init file and it also deserves a unique const.
I also agree w/ u that in the long term, it can/will be dropped, but I think dropping a single place constant will be easier than dropping a plain spread string (with optional whitespace in the middle).
Having said that, if you all agree to leave things as is, I can easily drop the commit.
@EdDev @0xFelix please let me know!
Thanks

Reply:
Well, `libvmi/cloudinit.go` includes machinery for building cloudinit specs, not to keep there some odd cloud-init commands.

I prefer not to have it in `libvmi` for the reasons expressed before.

Reply:
I suspect the usages are leftovers of old code and that it is not needed anymore.

But if you prefer to investigate it in a follow-up, you are welcome to place stuff like this in some other lib*.

I need to move `libvmi` to `pkg/` so it can be used by unit tests without importing e2e packages.
If we start adding stuff like these in the package, we will interfere to that move.
But to be honest, even without this, I think it is wrong. That "hello" has no meaning and I do not think we should encourage contributors to use it.

Reply:
ok! Dropped the commit and rebased it onto main!
Thank you!

Reply:
Yes, it is Cirros specific. This is why it is embedded in that factory.
 
 Most usages should use that factory.

Reply:
What about cases where Cirros VMs using DVs are constructed manually? Like here? https://github.com/kubevirt/kubevirt/pull/11448

Reply:
I guess that is an exception and that factory is not relevant. For such exceptions I do not think `libvmi`needs to create a factory, that specific test can do it.